### PR TITLE
Minor observable tweaks

### PR DESCRIPTION
**Pull request by stanieldev**
I changed a few IDs to match the previous ID changes that I had missed.

### CODE REVIEW

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java [hunk 1]
```diff
@@ -99,7 +99,9 @@ public static void generateMaterialBlocks() {
                         .setData(ProviderType.BLOCKSTATE, NonNullBiConsumer.noop())
                         .setData(ProviderType.LANG, NonNullBiConsumer.noop())
                         .setData(ProviderType.LOOT, NonNullBiConsumer.noop())
-                        //.tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.BEACON_BASE_BLOCKS)
+                        // TODO: Add #forge:storage_blocks tag
                         .color(() -> () -> MaterialBlock::tintedColor)
```
Comment (Yefancy) [inline common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java:105]: I'm not sure why it was commented before. Probably he has some ideas for tags @screret 
smth like adding tag for kjs's materials I think?

**Review by Yefancy — APPROVED** [review]
other changes are good for me

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java (hunk 1 above)
Comment (screret) [reply common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java:105]: We are injecting those tags runtime.
In TagLoaderMixin

Comment (Yefancy) [reply common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java:105]: so we do not need it anymore?

Comment (screret) [reply common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java:105]: Correct

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java [hunk 2]
```diff
@@ -99,7 +99,9 @@ public static void generateMaterialBlocks() {
                         .setData(ProviderType.BLOCKSTATE, NonNullBiConsumer.noop())
                         .setData(ProviderType.LANG, NonNullBiConsumer.noop())
                         .setData(ProviderType.LOOT, NonNullBiConsumer.noop())
-                        //.tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.BEACON_BASE_BLOCKS)
+                        // TODO: Add #forge:storage_blocks tag
```
Comment (Yefancy) [inline common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java:104]: merge it after you remove these lines and re-run datagen

**Review by Yefancy — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java (hunk 1 above)
Comment (stanieldev) [reply common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java:105]: What would you like removed from this section?

File: common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java (hunk 2 above)
Comment (stanieldev) [reply common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java:104]: Remove all 3 of the Green lines?

**Comment by stanieldev** [comment]
Think I did the rest. Should be ready for push now.

**Review by screret — APPROVED** [review]
lgtm

**Review by Yefancy — APPROVED** [review]

**Pull request merged**